helm-chart/renku-core/templates/deployment.yaml

```diff
@@ -27,6 +27,13 @@ spec:
         - name: shared-volume
           emptyDir: {}
       containers:
+        - name: {{ .Chart.Name}}-rqmetrics
+          image: "justsam/rqmetrics:latest"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: RQ_REDIS_HOST
+              value: {{ include "call-nested" (list . "redis" "redis.fullname") }}-master
+        
```

Comment:
does this need annotations to be picked up by prometheus?

Reply:
actually I guess they're needed in the service: https://github.com/SwissDataScienceCenter/renku-graph/blob/master/helm-chart/renku-graph/templates/event-log-service.yaml#L10-L13